src/classes/UTIL_Permissions_TEST.cls

```diff
@@ -47,6 +47,147 @@ public with sharing class UTIL_Permissions_TEST {
     private static String standardObjectName = 'Contact';
     private static String standardFieldName = 'FirstName';
 
+
+    @isTest
+    static void shouldReturnTrueForUserWithCreateAccessToObjectByString() {
+        System.runAs(getSysAdmin()) {
+            System.assert(UTIL_Permissions.canCreate(standardObjectName),
+                'Privileged user should have create access to standard object.');
+        }
+    }
+
+    @isTest
+    static void shouldThrowExceptionForUserWithoutCreateAccessToObjectByString() {
+        System.runAs(getReadOnlyUser()) {
+            try {
+                UTIL_Permissions.canCreate(standardObjectName);
+            } catch (Exception ex) {
+                System.assert(ex instanceof UTIL_Permissions.InsufficientPermissionException,
+                    'Minimum access user should not have create access to standard/custom object.');
+            }
+        }
+    }
+
+    @isTest
+    static void shouldReturnTrueForUserWithReadAccessToObjectByString() {
+        System.runAs(getSysAdmin()) {
+            System.assert(UTIL_Permissions.canRead(standardObjectName),
+                'Privileged user should have read access to standard object.');
+        }
+    }
+
+    @isTest
+    static void shouldThrowExceptionForUserWithoutReadAccessToObjectByString() {
+        System.runAs(getReadOnlyUser()) {
+            try {
+                UTIL_Permissions.canRead(standardObjectName);
+            } catch (Exception ex) {
+                System.assert(ex instanceof UTIL_Permissions.InsufficientPermissionException,
+                    'Minimum access user should not have read access to standard/custom object.');
+            }
+        }
+    }
+
+    @isTest
+    static void shouldReturnTrueForUserWithEditAccessToObjectByString() {
+        System.runAs(getSysAdmin()) {
+            System.assert(UTIL_Permissions.canUpdate(standardObjectName),
+                'Privileged user should have update access to standard object.');
+        }
+    }
+
+    @isTest
+    static void shouldThrowExceptionForUserWithoutEditAccessToObjectByString() {
+        System.runAs(getReadOnlyUser()) {
+            try {
+                UTIL_Permissions.canUpdate(standardObjectName);
+            } catch (Exception ex) {
+                System.assert(ex instanceof UTIL_Permissions.InsufficientPermissionException);
```

Comment:
Let's add a message to the assert, so it's consistent with the other methods.

Reply:
updated!